client/app/components/pages/manuscripts.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
+import { ManuscriptsService } from '../../services/manuscript.service';
+import { Manuscript } from '../../models/Manuscript';
+
+@Component({
+  moduleId: module.id,
+  providers:[],
+  selector: 'manuscripts',
+  templateUrl: '../../../../templates/manuscripts.component.html'
+})
+
+export class ManuscriptsComponent {
+	
+	private NewManuscript: String;
+	private newMan: Object;
+
+	constructor(private mScriptService: ManuscriptsService){
+		
```

Comment:
too many spaces across the page, spaces between the functions is usually enough.